OnLand/PDS/pod_sim.py

```diff
@@ -0,0 +1,50 @@
+import socket
+import time
+import random
+from PDS.config import POD_COMMANDER_PORT
+from PDS.UDP.Paradigm_pb2 import telemetry
+
+
+def create_telem(packet):
+    packet.lp1 = random.randint(1, 101)
+    packet.hp = random.randint(1, 101)
+    packet.sol1 = random.randint(0, 1)
+    packet.pressureVesselTemperature = random.randint(1, 101)
+    packet.railTemperature = random.randint(1, 101)
+    packet.sol2 = random.randint(0, 1)
+
+    return packet
+
+
+def main():
```

Comment:
Future work (make JIRA). Adding the flight config server, and possibly run the client UDP and server TCP connections in separate threads.